feat(clayui.com): Improve component docs by updating content Pt.2 (#3235)
liferay/clay: packages/clay-tooltip/README.mdx

<div class="nav-toc-absolute">
<div class="nav-toc">

-   [TooltipProvider](#tooltipprovider)
    -   [contentRenderer](#contentrenderer)
-   [API](#api)

</div>
</div>

> Simplest way of using Tooltip is by leveraging it's `show` prop and specifying `alignPosition` to determine it's position relative to the element it's tied to.

Reviewer: ```suggestion
Simplest way of using Tooltip is by leveraging it's `show` prop and specifying `alignPosition` to determine it's position relative to the element it's aligned to.
```

> By using the [TooltipProvider](#tooltipprovider) wrapper you can bring the control over the tooltip to the component that is tied to it.

Reviewer: ```suggestion
If you have multiple Tooltips on the page, [TooltipProvider](#tooltipprovider) allows you to only have to instantiate the component once and simply provide html attributes to the elements that need tooltips.
```

<Tooltip />

## TooltipProvider

By using the `<TooltipProvider>` it allows you to add the desired tooltip content on each element as a data attribute.

> Here's a list of attributes the TooltipProvider can leverage:

Reviewer: ```suggestion
Here's a list of html attributes that you can provide to children elements of the TooltipProvider:
```
